scripts/content.js

```diff
@@ -1,36 +0,0 @@
-let currentPage;
```

Comment:
This file was now unnecessary - the service worker is doing all of the work -  so I removed it.